### DISCUSSION TITLE
Папка '/web' не обнаружена. Свободно памяти: 304996

### DISCUSSION THREAD

**Discussion by yusufsulaiman** [vvip-68/LedPanelWiFi#4] [Вопросы и ответы] [answered]
привет.
Я успешно скомпилировал
после того, как я подключился к сети PANELAP
и подключение к 192.168.4.1
у меня сервер не активен
после того, как я посмотрел сериал, я не нашел веб-папку.
я загрузил данные
Что мне нужно сделать
Спасибо

**Comment by vvip-68** [comment, marked as answer]
https://github.com/vvip-68/LedPanelWiFi/wiki/Пошаговая-инструкция-подготовки-среды-и-сборка-проекта#шаг-10
https://github.com/vvip-68/LedPanelWiFi/wiki/Загрузка-данных-в-файловую-систему-МК

**Reply by yusufsulaiman** [reply]
Спасибо
я преуспел
